ui/webui/src/components/storage/DiskEncryption.jsx

```diff
@@ -91,6 +91,34 @@ const getStrengthLevels = (minQualility, isStrict) => {
     return levels;
 };
 
+const rules = [
+    {
+        id: "length",
+        text: (policy) => cockpit.format(_("Must be at least $0 characters"), policy["min-length"].v),
+        check: (policy, password) => password.length >= policy["min-length"].v,
+        isWarning: false,
```

Comment:
I think it's more explicit to use `isError: false` instead of `isWarning: true`. and vice versa. I had to read the code through to understand that `isWarning: true` is not causing the password to be invalid.

Reply:
Updated